### ISSUE TITLE
Issue with RN 0.44.2

### ISSUE THREAD

**Issue by ShMcK** [react-native-community/react-native-svg#346]
I'm unable to update react-native-svg to the latest react native 0.44.2. See the following error:

```
Unable to resolve module `AccessibilityInfo` from `PATH/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map or in the directories:
PATH/node_modules/react-native/node_modules,
PATH/node_modules
```

**Comment by ShMcK** [comment]
No longer an issue with 0.44.3

**Comment by sdeleon28** [comment]
0.45.1 fixes this as well. `react-native-git-upgrade` did the trick for me.

**Comment by mateioprea** [comment]
Apparently it's not fixed. I get this error everytime: 
ld: library not found for -lRNSVG-tvOS
clang: error: linker command failed with exit code 1 (use -v to see invocation)

mateioprea$ react-native --version
react-native-cli: 2.0.1
react-native: 0.45.1

If I unlink the library everything seems fine.
